core/client-internal/src/test/java/tachyon/client/file/BaseFileSystemTest.java

```diff
@@ -69,7 +69,6 @@
    */
   @Before
   public void before() {
-    ClientContext.reset();
```

Comment:
in `BaseFileSystemTest`, `FileSystemContext` has been modified, should we reset ClientContext in `after`?